components/FeatureHighlight/src/private/MDCFeatureHighlightDismissGestureRecognizer.m

```diff
@@ -86,7 +86,11 @@ - (void)touchesEnded:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
   [super touchesEnded:touches withEvent:event];
 
   if (_hasTouch) {
-    self.state = UIGestureRecognizerStateEnded;
+    if (MDCCGFloatEqual(_progress, 1.0)) {
```

Comment:
Could you add a comment explaining when _progress == 1.0, and why we want to cancel in that case?